lib/utils/disk.go

```diff
@@ -125,3 +126,20 @@ func CanUserWriteTo(path string) (bool, error) {
 
 	return false, nil
 }
+
+// FreeDiskWithReserve returns the available disk space.
+func FreeDiskWithReserve(dir string, reservedFreeDisk uint64) (uint64, error) {
+	var stat unix.Statfs_t
+	err := unix.Statfs(dir, &stat)
```

Comment:
Looks like this is identical to the version in the syscall package used by PercentUsed. Might be better to use the standard library instead of /x/sys/unix, if only for consistency.